learningresources/tests/test_case.py

```diff
@@ -0,0 +1,29 @@
+"""
+A subclass of the Django TestCase which handles
+often-needed features, like creating an authenticated
+test client.
+"""
+
+from django.test import Client
+from django.test.testcases import TestCase
+from django.contrib.auth.models import User
+
+
+class LoreTestCase(TestCase):
+    """Handle often-needed things in tests."""
+
+    def setUp(self):
+        """set up"""
+        super(LoreTestCase, self).setUp()
+        self.username = 'test'
+        self.password = 'test'
+        self.user = User.objects.create(username=self.username)
+        self.user.set_password(self.password)
+        self.user.save()
+
+        self.client = Client()
+        self.client.login(username=self.username, password=self.password)
+
+    def tearDown(self):
```

Comment:
Why do we need this if it is just calling super? I think you can either remove this, or add a deletion statement for the user we created in setup if we want to be super explicit maybe?  Your call